discord/websocket.py

```diff
@@ -65,7 +65,7 @@ def on_websocket_message(self, msg: dict) -> dict:
         # check if the last four bytes are equal to ZLIB_SUFFIX
         if len(msg) < 4 or msg[-4:] != b'\x00\x00\xff\xff':
             self.buffer = bytearray()
-            return msg.decode('utf-8')
+            return
```

Comment:
U cannot decode a binary message without decompressing it. In your code it just randomly decides to return an invalid payload if you read the docs it's pretty clear that if the zlib suffix doesn't match it's not valid and if the zlib suffix matches then u need to decompress it and then decode it.

I'm already working on my own lib, I dont need to look at the dpy code.

Reply:
In that case, I would appreciate if you joined the dev team.
Since in the end, it really doesn't matter who owns the lib IMO
What matters is that we unite the community under one single wrapper which is better